nvdaHelper/remote/ia2LiveRegions.cpp

```diff
@@ -193,6 +193,34 @@ bool getTextFromIAccessible(wstring& textBuf, IAccessible2* pacc2, bool useNewTe
 	return gotText;
 }
 
+bool checkAccessibleStateAndRole(IAccessible* pacc, VARIANT varChild) {
+	//Retrieve the object states, and if its invisible or offscreen ignore the event.
+	VARIANT varState, varRole;
+	// #7709: Ensure The VARIANT is initialized, as accState may try to clear it before setting it.
+	VariantInit(&varState);
+	pacc->get_accState(varChild,&varState);
+	if(varState.vt==VT_I4&&(varState.lVal&STATE_SYSTEM_INVISIBLE)) {
+		VariantClear(&varState);
+		return false;
+	}
+	VariantClear(&varState);
+	//Retrieve the object role and ignore ROLE_SYSTEM_STATICTEXT unless it's a hyperlink. There will be
+	//an event on its parent which handles this better.

```

Comment:
Could you change this comment to:
// return false for text leaf nodes as their text is available by their parents' IAccessibleText interface.
// Text leaf nodes have a role of staticText and do not support the IAccessibleHyperlink interface (I.e. they are not represented by an embedded object character on their parent).
// #8044: Chrome fires redundant events on these nodes which would cause us to double speak live regions if not ignored.

Reply:
Done.